package.json

```diff
@@ -29,8 +29,8 @@
     "cordova-fetch": "^2.0.0",
     "fs-extra": "^8.1.0",
     "import-fresh": "^3.1.0",
-    "is-url": "^1.2.4",
     "isobject": "^4.0.0",
+    "npm-package-arg": "^6.1.1",
```

Comment:
dumb question: do we need to explicitly install this package or can we just require it, because it is bundled with npm per default?

Reply:
It needs to be installed otherwise you'll receive a `MODULE_NOT_FOUND` error when you try to require it.

Reply:
Thx! :)